Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp

```diff
@@ -276,7 +276,7 @@ void CEXIMemoryCard::TransferComplete()
 void CEXIMemoryCard::CmdDoneLater(u64 cycles)
 {
   CoreTiming::RemoveEvent(s_et_cmd_done[card_index]);
```

Comment:
Maybe not worthwhile to do this here but I was thrown off by the fact this was actually a member variable.  Looks like none of the class members is prefixed with `m_`

Reply:
Oh yeah, that should probably be fixed... I'll fix the enum in the header as well.